examples/brotli/boo.cpp

```diff
@@ -0,0 +1,5 @@
+#include <brotli/decode.h>
+#include <brotli/encode.h>
+
+int main() {
```

Comment:
please use something (if possible) from the library to check linking as well as include paths

Reply:
I've used some basic functions to check the version of library, just to check linking.

Reply:
Nice 👍